src/test/shell/integration/toolchain_test.sh

```diff
@@ -708,9 +708,9 @@ EOF
   expect_log "Performing resolution of //${pkg}/toolchain:test_toolchain for target platform ${default_host_platform}"
   expect_log "Rejected toolchain //${pkg}/demo:toolchain_impl_invalid; mismatching config settings: optimised"
   expect_log "Toolchain //register/${pkg}:test_toolchain_impl_1 is compatible with target platform, searching for execution platforms:"
-  expect_log "Compatible execution platform ${default_host_platform_alias}"
+  expect_log "Compatible execution platform ${default_host_platform}"
```

Comment:
very nice! could you also remove https://cs.opensource.google/bazel/bazel/+/master:src/test/shell/testenv.sh.tmpl;drc=905a4fe7cb2412bd3a03a857266e824aaa49bfd1;l=96 ?

Reply:
Done. I had thought there was another use, but you're right, that was all.